src/senaite/api/__init__.py

```diff
@@ -350,8 +350,11 @@ def get_url(brain_or_object):
     :returns: Absolute URL
     :rtype: string
     """
-    if is_brain(brain_or_object) and base_hasattr(brain_or_object, "getURL"):
-        return brain_or_object.getURL()
+    if is_brain(brain_or_object) and base_hasattr(brain_or_object, "getPath"):
+        portal = get_portal()
+        portal_path = portal.absolute_url()
+        ret = '{}/{}'.format(portal_path, brain_or_object.getPath())
+        return ret
```

Comment:
Why do you need that Nihad?
This will cause problems when it comes to virtual hosting, e.g. then you get instead of https://senaite.ridingbytes.com/front-page -> https://senaite.ridingbytes.com/Plone/front-page.

Reply:
oh I see :( 
The thing is I was getting URLs as 'http://localhost:8080/bika_setup/bika_referencedefinitions/referencedefinition-3' which misses 'Plone' there. 
So how can we handle both cases in the shortest way? Checking if URL string contains `Site Name` sounded dangerous to me.